pkg/replica/replica.go

```diff
@@ -232,6 +236,37 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 	return r, r.writeVolumeMetaData(true, r.info.Rebuilding)
 }
 
+func (r *Replica) checkVolumeMetaFile() (err error) {
```

Comment:
Suggest separating the `checkVolumeMetaFile` to tw functions `checkVolumeMetaFile` and `createVolumeMetaFile`. Have the functions more clear and specific.

Reply:
Modified. And changed the method name.